### PR TITLE
[Nikhil] - Sage - How to Create your own DAO on Celo

### PR DESCRIPTION
**Pull request by nikbhintade**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

**Comment by nikbhintade** [comment]
Hi guys, I have made sure everything is correct but let me know if there is anything I may have missed.

File: blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+contract Token is ERC20 {
+   
+    constructor() ERC20("Token", "TOKEN") {
+        _mint(msg.sender, 1000000);
+    }
+ 
+}
+```
+
+Please go here to read more about how the above snippet works. We can now go to the smart contract for the DAO.
```
Comment (viral-sangani) [inline blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md:76]: Is there supposed to be a URL here?

File: blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+DAO Smart Contract
```
Comment (viral-sangani) [inline blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md:77]:
```suggestion

#### DAO Smart Contract
```

File: blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+        uint256 reject;
+        uint256 abstain;
+        uint256 deadline;
+        bytes32 title;
+        mapping(address => bool) voted;
+    }
+```
+
+> Note: Everything other than functions goes above the constructor, and all the functions go below the constructor. Just a tip on organizing smart contracts
+Storing Created Proposals
```
Comment (viral-sangani) [inline blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md:123]:
```suggestion

### Storing Created Proposals
```

File: blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+
+We need to store any proposal that is made. By creating a mapping called `proposals`, we will achieve this. The proposal will serve as the value, and the proposal's index, which we will track using the `proposalIndex` variable, will be the key in the mapping.
+
+```js
+    uint256 public proposalIndex;
+    mapping(uint256 => Proposal) public proposals;
+```
+Create Proposal
```
Comment (viral-sangani) [inline blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md:131]:
```suggestion

### Create Proposal
```

**Review by viral-sangani — CHANGES_REQUESTED** [review]

**Comment by viral-sangani** [comment]
Hey @nikbhintade Nice work. I have made few suggestions, other than that, this look good to me.

File: blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md (hunk 1 above)
Comment (nikbhintade) [reply blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md:76]: Yeah. There is a url. I will add it.

File: blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md (hunk 2 above)
Comment (nikbhintade) [reply blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md]: I have made this heading 1.

File: blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md (hunk 3 above)
Comment (nikbhintade) [reply blog/12-18-22-celo-dao/12-18-22-how-to-create-dao-on-celo.md]: For keeping formatting consistent, I have made this heading 2.

**Review by viral-sangani — APPROVED** [review]

**Pull request merged**